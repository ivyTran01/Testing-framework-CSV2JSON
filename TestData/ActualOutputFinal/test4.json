[
 {
  "field1":"Ivy T.",
  "field2":22,
  "field3":true,
  "field4":"5.2 \"inside\" end"
 }
]