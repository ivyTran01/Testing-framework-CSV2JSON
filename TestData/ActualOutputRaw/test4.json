[
{"field1":"Ivy","field2":22,"field3":true,"field4":"5.2 \"inside\" end"}
]
