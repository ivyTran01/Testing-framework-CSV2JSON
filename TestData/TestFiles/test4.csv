Ivy;22;true
"";25;false
Danny;21;true
