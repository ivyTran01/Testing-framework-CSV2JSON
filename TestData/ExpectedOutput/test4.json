[{"field1":"Ivy","field2":22,"field3":true},{"field1":null,"field2":25,"field3":false},{"field1":"Danny","field2":21,"field3":true}]